### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by boyinggong**
add confusion matrix and ROC curve

### CODE REVIEW

File: code/utils/logistic_function.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+def plot_roc(logreg_proba, y):
```
Comment (briankleinqiu) [inline code/utils/logistic_function.py]: what's the y input here?

**Comment by BenjaminHsieh** [comment]
I believe the y are the actual responses (0 or 1), @boyinggong can you make some comments to the function

**Comment by boyinggong** [comment]
Sorry I've updated the comments.

**Comment by BenjaminHsieh** [comment]
Check #112

**Comment by boyinggong** [comment]
@BenjaminHsieh I've combined the code into the logistic_function.py and logistic.py file. I also update the code to subplot all figures into one. Please check it and let me know if there is any error.

**Comment by BenjaminHsieh** [comment]
Cool, can you also add a subject label at the top of each ROC graph?

**Comment by BenjaminHsieh** [comment]
I am going to merge this for now, we can add the subject label later

**Pull request merged**